airbyte-integrations/connectors/source-elasticsearch/source_elasticsearch/source.py

```diff
@@ -0,0 +1,179 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+import json
+from datetime import datetime
+from typing import Dict, Generator
+
+from airbyte_cdk.logger import AirbyteLogger
+from airbyte_cdk.models import (
+    AirbyteCatalog,
+    AirbyteConnectionStatus,
+    AirbyteMessage,
+    AirbyteRecordMessage,
+    AirbyteStream,
+    ConfiguredAirbyteCatalog,
+    Status,
+    Type,
+)
+from airbyte_cdk.sources import Source
+from elasticsearch import Elasticsearch
+
+
+class UnsupportedDataTypeException(Exception):
+    pass
+
+
+class SourceElasticsearch(Source):
+    system_indices = [
+        ".kibana_1",
+        ".opendistro_security",
```

Comment:
I don't think this is the only system indices, should be better to have a specification option to read only the indices user want to.